custom_components/waste_collection_schedule/waste_collection_schedule/source/abfall_neunkirchen_siegerland_de.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import requests
+from waste_collection_schedule import Collection
+from waste_collection_schedule.service.ICS import ICS
+
+TITLE = "Abfall Neunkirchen Siegerland"
+DESCRIPTION = " Source for 'Abfallkalender Neunkirchen Siegerland'."
+URL = "https://www.neunkirchen-siegerland.de"
+TEST_CASES = {
+
```

Comment:
Please add at least one test case (any valid address) and run `test_sources.py -s abfall_neunkrichen_siegerland_de`.

Reply:
Testcase added

output of test_sources.py:

 python3 test_sources.py -s abfall_neunkirchen_siegerland_de
Testing source abfall_neunkirchen_siegerland_de ...
  found 91 entries for Waldstraße